db/migrate/20181019072511_deduplicate_esfa_subscriber_list.rb

```diff
@@ -0,0 +1,30 @@
+class DeduplicateEsfaSubscriberList < ActiveRecord::Migration[5.2]
+  def change
+    old_list = SubscriberList.find_by(
+      slug: "education-and-education-and-skills-funding-agency"
+    )
+    new_list = SubscriberList.find_by(
+      slug: "education-and-skills-funding-agency"
+    )
+
+    if old_list && new_list
+      puts "#{old_list.subscriptions.count} subscriptions for '#{old_list.title}'"
+      puts "#{new_list.subscriptions.count} subscriptions for '#{new_list.title}'"
+      puts "#{(old_list.subscribers & new_list.subscribers).count} subscribers exist in both lists"
+
+      scope = Subscription
+        .where(subscriber_list: old_list)
+        .where.not(subscriber: new_list.subscribers)
+
+      puts "#{scope.count} subscriptions to migrate from '#{old_list.title}' to '#{new_list.title}'"
+
+      updated = scope.update_all(subscriber_list_id: new_list.id)
+
+      puts "Added #{updated} subscriptions to '#{new_list.title}'"
```

Comment:
Did you want to display `updated` or `updated.count` here?

Reply:
@binaryberry `update_all` returns the number of affected records so this produced the output `Added 1528 subscriptions to 'Education and Skills Funding Agency'` on integration